.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
```

Comment:
make runs unit tests, do we need a separate unit-test target? Oh, I didn't know about the exit code, does && actually do a boolean and on the exit code?